frictionless/server/file/create.py

```diff
@@ -17,12 +17,10 @@ class Result(BaseModel):
 @router.post("/file/create")
 async def server_file_create(
```

Comment:
@shashigharti 
OK, the problem is here -- `/file/create` endpoint should be just a normal JSON endpoint (see the way others endpoints are written with `Props/Result` data structures)

The reason why `/file/upload` is written differently to other endpoint is because it needs to handle file uploading - https://fastapi.tiangolo.com/tutorial/request-files/

Also note, that for file uploading client's `makeRequest` does some magic with multipart form

Reply:
Thank you!